custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json

```diff
@@ -16,15 +16,65 @@
     "referenced_doc_type": "CommCareCase",
     "description": "",
     "base_item_expression": {
-      "test": {},
+      "test": {
+        "type": "and",
+        "filters": [
+          {
+            "operator": "eq",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "root_doc",
+              "expression": {
+                "datatype": null,
+                "type": "property_name",
+                "property_name": "closed"

```

Comment:
Do you forsee any issues with the aggregation from not having rows for closed cases?